### DISCUSSION TITLE
Does this also work when you change the default login of the inverters?

### DISCUSSION THREAD

**Discussion by DaKhalli** [StephanJoubert/home_assistant_solarman#256] [Q&A]
I changed the default pasword of my inverters. I think thats the reason why this plugin is not working.

Is there a way to work without the default admin/admin passwords?

I think its a weak security to leave the default passwords, but i would like homeassist to collect the solar data.

Thanks in advance (ps, using deye microinverters, they show up and work in de solarman app)

**Comment by tianbr** [comment]
I am not using the default username and password on my Deye inverter and the integration still works.
